ert/storage/_storage.py

```diff
@@ -99,28 +110,109 @@ async def _load_blob_record(self) -> ert.data.BlobRecord:
         return ert.data.BlobRecord(data=record.data)
 
 
+async def get_record_storage_transmitters(
+    records_url: str,
+    record_name: str,
+    record_source: Optional[str] = None,
+    ensemble_size: Optional[int] = None,
+) -> Dict[int, Dict[str, StorageRecordTransmitter]]:
+    if record_source is None:
+        record_source = record_name
+    uri = f"{records_url}/{record_source}"
+    metadata = await get_record_metadata(uri)
+    record_type = metadata["record_type"]
+    uris = metadata["uris"]
+    # We giving an ensemble size we expect the number of uris in the record metadata
+    # to match the size of the ensemble or be equal to 1, in the case of an
+    # uniform record and, has the same data stored only once for all the realizations
+    if ensemble_size is not None and 1 < len(uris) != ensemble_size:
+        raise ert.exceptions.ErtError(
+            f"Ensemble size {ensemble_size} does not match stored record ensemble size {len(uris)}"
+        )
+
+    transmitters = []
+    for record_uri in uris:
+        transmitter = StorageRecordTransmitter(record_source, records_url)
+        # Record data has already been stored, now just setting the transmitter uri and record type
+        transmitter.set_transmitted(record_uri, record_type)
+        transmitters.append(transmitter)
+
+    if ensemble_size is not None and len(transmitters) == 1:
+        return {iens: {record_name: transmitters[0]} for iens in range(ensemble_size)}
+    return {
+        iens: {record_name: transmitter}
+        for iens, transmitter in enumerate(transmitters)
+    }
+
+
+def get(url: str, headers: Dict[str, Any]) -> requests.Response:
+    return requests.get(url, headers=headers)
+
+
 async def _get_from_server_async(
     url: str,
     headers: Dict[str, str],
     **kwargs: Any,
-) -> httpx.Response:
-    async with httpx.AsyncClient() as session:
-        resp = await session.get(url=url, headers=headers, timeout=None, **kwargs)
+) -> requests.Response:
```

Comment:
Was it decided to not use httpx now?  It is still in the set.py and release notes, but apparently not used anywhere.

Reply:
I decided not to use it anymore due to the intermittent errors that appear when dealing with large realizations mainly in the ert3 polynomial example test.

I will remove it from the setup.py also